frontend/src/component/releases/ReleasePlanTemplate/MilestoneStrategyTitle.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Typography } from '@mui/material';
+import Input from 'component/common/Input/Input';
+
+interface IMilestoneStrategyTitleProps {
+    title: string;
+    setTitle: (title: string) => void;
+}
+
+export const MilestoneStrategyTitle = ({
+    title,
+    setTitle,
+}: IMilestoneStrategyTitleProps) => {
+    return (
+        <Box sx={{ paddingBottom: (theme) => theme.spacing(2) }}>
+            <Typography
+                sx={{
+                    paddingBottom: (theme) => theme.spacing(2),
+                }}
```

Comment:
We usually prefer to declare styled components over using `sx`, but this is not the worst offender, so I'll leave it up to you. Also, when using `sx` I think you could simplify this to `paddingBottom: 2`

Reply:
Fixed, there was a Box as well so made them both Styled*